packages/react-components/react-provider/package.json

```diff
@@ -37,6 +37,7 @@
     "@fluentui/react-tabster": "^9.12.2",
     "@fluentui/react-theme": "^9.1.11",
     "@fluentui/react-utilities": "^9.11.1",
+    "@fluentui/react-icons": "^2.0.207",
```

Comment:
_nit:_ Move this to the top of the dependencies so they are ordered alphabetically